components/breadcrumb/css/_mixin.scss

```diff
@@ -32,7 +33,12 @@
 }
 
 @mixin utrecht-breadcrumb__item {
+  align-items: baseline;
   block-size: 100%;
+  display: inline-flex;
+  & > * {
+    margin-inline-end: var(--utrecht-breadcrumb-item-gap, 0);
+  }
```

Comment:
If this is to remove a margin on link, I would rather have the property on `__link`.